Gemfile

```diff
@@ -34,6 +34,8 @@ end
 group :test do
   gem 'cucumber-rails', require: false
   gem 'database_cleaner'
+  gem 'factory_girl_rails', '~> 4.5'
+  gem 'mailcatcher'
```

Comment:
Remember when I said to just include mailcatcher. Yeah, I changed my mind. Can you remove it and add a note in the README?